patches/react-native-haptic-feedback+1.11.0.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/react-native-haptic-feedback/android/build.gradle b/node_modules/react-native-haptic-feedback/android/build.gradle
+index c326a26..dd33c61 100644
+--- a/node_modules/react-native-haptic-feedback/android/build.gradle
++++ b/node_modules/react-native-haptic-feedback/android/build.gradle
+@@ -1,6 +1,7 @@
+ def DEFAULT_COMPILE_SDK_VERSION             = 27
+ def DEFAULT_BUILD_TOOLS_VERSION             = "27.0.0"
+ def DEFAULT_TARGET_SDK_VERSION              = 27
++def DEFAULT_MIN_SDK_VERSION                 = 16
+ 
+ buildscript {
+     repositories {
+@@ -20,7 +21,7 @@ android {
+     buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : DEFAULT_BUILD_TOOLS_VERSION
+ 
+     defaultConfig {
+-        minSdkVersion 16
++        minSdkVersion rootProject.hasProperty('minSdkVersion') ? rootProject.minSdkVersion : DEFAULT_MIN_SDK_VERSION
```

Comment:
this was a problem in bitrise, even though we can build locally with minSdkVersion 16?

Reply:
Yeh, it seems this is necessary to change. And there is actually a PR in their repository to update this. Just not pushed into the release yet.